From af8658036325cfd4810fdffea3b94c4edee66088 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Thu, 24 Jan 2019 16:36:19 +0000
Subject: [PATCH 253/587] staging: bcm2835-codec: Fix potentially uninitialised
 vars

src_m2m_buf and dst_m2m_buf were printed in log messages
when there are code paths that don't initialise them.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 .../staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index 4ef16379f216..c26760faae2d 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -743,7 +743,7 @@ static void device_run(void *priv)
 	struct bcm2835_codec_ctx *ctx = priv;
 	struct bcm2835_codec_dev *dev = ctx->dev;
 	struct vb2_v4l2_buffer *src_buf, *dst_buf;
-	struct m2m_mmal_buffer *src_m2m_buf, *dst_m2m_buf;
+	struct m2m_mmal_buffer *src_m2m_buf = NULL, *dst_m2m_buf = NULL;
 	struct v4l2_m2m_buffer *m2m;
 	int ret;
 
-- 
2.17.1

